### DISCUSSION TITLE
Jaeger service from Istio states service no available

### DISCUSSION THREAD

**Discussion by OteemoSanjay** [kiali/kiali#7410] [Q&A]
### Describe the bug

We have a self managed k8 cluster with Istio (1.22) and Kiali enabled.
When we click on the Jaeger link, we get the following error.

{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "no service port 16686 found for service \"tracing\"",
  "reason": "ServiceUnavailable",
  "code": 503
}

When I opened this issue as a user experience issue in ISTIO, they suggested I open it in KIALI

### What are the steps to reproduce this bug?

1. Open Rancher UI
2. Click Istio 
3. Open Jaeger page

### Environment
*Learn about how to determine versions [here](https://kiali.io/docs/faq/general/#how-do-i-determine-what-version-i-am-running).*

- **Kiali version : 1.76.0
- ** Istio version 1.22.0

❯ istioctl version
client version: 1.22.1
control plane version: 1.22.1
data plane version: 1.22.1 (4 proxies)

- **Kubernetes impl:
- RKE cluster with a Rancher UI for the UI interface.
- **Kubernetes version: 1.28.10
- **Other notable environmental factors:**

Istio and Kiali have been working fine for a almost a year, but someone wanted distributed tracing.
Even though the error says service no available, the service YAML is within the large Jaeger YAML

```
apiVersion: v1
kind: Service
metadata:
  name: tracing
  namespace: istio-system
  labels:
    app: jaeger
spec:
  type: ClusterIP
  ports:
    - name: http-query
      port: 80
      protocol: TCP
      targetPort: 16686
  selector:
    app: jaeger
``

But I think in the Jaeger ALL in ONE implementation, there is no port for the service to invoke

**Comment by jmazzitelli** [comment]
_converting to Q&A as this is likely not a bug_

Please consult the Kiali docs on configuring Jaeger tracing - it sounds like you just need to configure Kiali correctly to be able to communiate with Jaeger

https://kiali.io/docs/configuration/p8s-jaeger-grafana/tracing/jaeger/

**Reply by jmazzitelli** [reply]
And here is the tracing section in the Kiali docs FAQ that may also help:

https://kiali.io/docs/faq/distributed-tracing/

**Reply by OteemoSanjay** [reply]
Will it may be a bug if the jaeger YAML shipped with Istio doesn't have the port / or targetPort defined
